dsl_parser/tasks.py

```diff
@@ -68,10 +68,9 @@ def prepare_deployment_plan(plan, inputs=None, **kwargs):
     """
     Prepare a plan for deployment
     """
-    plan = multi_instance.create_deployment_plan(plan)
```

Comment:
multi_instance.create_deployment_plan clones nodes depending on number of instances we request in 'deploy' property. It expect that property holds integer so in our case if we allow also intrinsic we have to first evaluate it by calling _process_functions. I didn't find any potential problems with reordering of calls of that functions so I just did it.